src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleRouter.cs

```diff
@@ -0,0 +1,148 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web;
+using AngleSharp.Io;
+using EnsureThat;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Routing;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Routing.Template;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Api.Features.ActionConstraints;
+using Microsoft.Health.Fhir.Core.Features;
+
+namespace Microsoft.Health.Fhir.Api.Features.Resources.Bundle
+{
+    internal class BundleRouter : IRouter
```

Comment:
It would be great to have a description mentioning how this router is used, and why it's needed.